msteams_hermes.gemspec

```diff
@@ -16,12 +16,13 @@ Gem::Specification.new do |spec|
                        "Microsoft's adaptive cards."
   spec.homepage      = "https://github.com/xing/msteams_hermes"
   spec.license       = "MIT"
-  spec.required_ruby_version = Gem::Requirement.new(">= 2.7.0")
 
   spec.metadata["homepage_uri"] = spec.homepage
   spec.metadata["source_code_uri"] = "https://github.com/xing/msteams_hermes"
   spec.metadata["changelog_uri"] = "https://github.com/xing/msteams_hermes/CHANGELOG.md"
 
+  spec.required_ruby_version = Gem::Requirement.new(">= 3.3")
```

Comment:
Do you think we should raise the semantic version string "more" that just the patch version because we basically drop support for ruby 2.x.x and even < 3.3 ?

Reply:
Since I did some refactoring related to `Values in Hash literals and keyword arguments can be omitted.` (from https://www.ruby-lang.org/en/news/2021/12/25/ruby-3-1-0-released)
I was concerned about the compatibility.
But it seems that is from ruby `3.1` onwards, so I adapted it now